index.bs

```diff
@@ -481,15 +491,35 @@ regardless of the <a>browsing context</a>.
 
 ## Permissions Policy ## {#permissions-policy}
 
-This specification defines a <a>policy-controlled feature</a> named
-`"picture-in-picture"` that controls whether the <a>request Picture-in-Picture
-algorithm</a> may return a {{SecurityError}} and whether
-{{pictureInPictureEnabled}} is `true` or `false`.
+This specification defines two <a>policy-controlled features</a> that control
+whether picture-in-picture is allowed:
+
+<dl dfn-type=value>
+  <dt>"<dfn permission export>picture-in-picture</dfn>".
+  <dd>
+    The `"picture-in-picture"` permission policy.
```

Comment:
This is tautological, as it's not actually defining what the policy *does* and its default allow list. The definition here should state what the policy is for in a descriptive manner.  Or, alternatively (or concretely) link to the algorithm where this algorithm comes into play.